package-lock.json

```diff
@@ -1397,28 +1411,22 @@
       }
     },
     "follow-redirects": {
-      "version": "1.7.0",
-      "resolved": "https://registry.npmjs.org/follow-redirects/-/follow-redirects-1.7.0.tgz",
-      "integrity": "sha512-m/pZQy4Gj287eNy94nivy5wchN3Kp+Q5WgUPNy5lJSZ3sgkVKSYV/ZChMAQVIgx1SqfZ2zBZtPA2YlXIWxxJOQ==",
+      "version": "1.5.10",
```

Comment:
Why does this now use an older version of the library?

Reply:
/shrug this file is autogenerated by NPM

Reply:
because https://github.com/axios/axios/commit/2eeb59af4de41e07c7dd1eec09af0230960c029c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2